From 3800a61d939385346606cd70b56a005c62cb49df Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Mon, 9 Jan 2023 10:56:03 +0100
Subject: [PATCH 328/529] ARM: dts: stm32: add adc pins muxing on
 stm32mp135f-dk

Define pins muxing that can be used for ADC on stm32mp135f-dk board
(USB Type-C CC1 & CC2 pins).

Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/283315
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Signed-off-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Change-Id: I45b93bce55d31482be04eb7dae20d17183e0e420
---
 arch/arm/boot/dts/stm32mp13-pinctrl.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/arm/boot/dts/stm32mp13-pinctrl.dtsi
+++ b/arch/arm/boot/dts/stm32mp13-pinctrl.dtsi
@@ -6,6 +6,13 @@
 #include <dt-bindings/pinctrl/stm32-pinfunc.h>
 
 &pinctrl {
+	adc1_usb_cc_pins_a: adc1-usb-cc-pins-0 {
+		pins {
+			pinmux = <STM32_PINMUX('F', 12, ANALOG)>, /* ADC1 in6 */
+				 <STM32_PINMUX('A', 3, ANALOG)>; /* ADC1 in12 */
+		};
+	};
+
 	dfsdm_clkout_pins_a: dfsdm-clkout-pins-0 {
 		pins {
 			pinmux = <STM32_PINMUX('C', 3, AF3)>; /* DFSDM_CKOUT */
